system/Router/RouteCollection.php

```diff
@@ -150,7 +150,7 @@ class RouteCollection implements RouteCollectionInterface
     /**
      * The current method that the script is being called by.
      *
-     * @var string
+     * @var string HTTP verb (lower case) or `*`
```

Comment:
can this include possible verb value? eg: `get|post`...

Reply:
Added.